src/components/SetGuestUsername.vue

```diff
@@ -170,4 +209,10 @@ export default {
 	}
 }
 
+.modal__content {
+	width: 50vw;
+	margin: 10vw 0;
+	text-align: center;
+}
```

Comment:
Something missing from the Modal component?

Reply:
It was unnecessary. Thank you.